picard/formats/asf.py

```diff
@@ -186,8 +186,11 @@ def _save(self, filename, metadata):
         log.debug("Saving file %r", filename)
         file = ASF(encode_filename(filename))
 
-        if config.setting['clear_existing_tags']:
-            file.tags.clear()
+        if config.setting["clear_existing_tags"]:
```

Comment:
Hmmm, what about going the pythonic way, try a method, and if it fails try another one ?